src/test/gen/block_gen.h

```diff
@@ -0,0 +1,58 @@
+#ifndef BITCOIN_TEST_GEN_BLOCK_GEN_H
+#define BITCOIN_TEST_GEN_BLOCK_GEN_H
+
+#include "test/gen/crypto_gen.h"
+#include "test/gen/transaction_gen.h"
+#include "uint256.h"
+#include "primitives/block.h"
```

Comment:
Please use bracket syntax includes ("#include <foo.h>") instead of quote syntax includes: